### PR TITLE
ModularServer: Always set model.running = True on reset

### PR DESCRIPTION
**Pull request by rht**
Addresses https://github.com/projectmesa/mesa/discussions/1397.
- [x] tested.
~Do not merge yet~ can merge now.

### CODE REVIEW

**Comment by rht** [comment]
OK I have tested it. @eeyouol, with this PR getting merged, you no longer have to specify the `self.running = True`.

And I don't have to update the advanced tutorial.
Sometimes, the best documentation is "no documentation needed".

File: mesa/visualization/ModularVisualization.py [hunk 1]
```diff
@@ -341,6 +341,9 @@ def reset_model(self):
                 model_params[key] = val
 
         self.model = self.model_cls(**model_params)
+        # We specify the `running` attribute her so that the user doesn't have
```
Comment (tpike3) [inline mesa/visualization/ModularVisualization.py:344]: should be here not her...but otherwise looks good

Comment (rht) [reply mesa/visualization/ModularVisualization.py]: Ugh, fixed!

**Review by tpike3 — APPROVED** [review]
LGTM

**Pull request merged**